src/main/java/seedu/duke/commands/ExitCommand.java

```diff
@@ -0,0 +1,15 @@
+package seedu.duke.commands;
+
+import seedu.duke.constants.MessageConstants;
+import seedu.duke.entrylog.EntryLog;
+
+public class ExitCommand extends Command {
+    public ExitCommand() {
+        this.isExit = true;
+    }
+
+    @Override
+    public void execute(EntryLog entries) {
+        ui.print(MessageConstants.MESSAGE_EXIT);
```

Comment:
The UI class has the `printExit()` method implemented. It should be used to standardise the formatting of messages output by the application. Using `print()` directly will lead to formatting issues when the application exits.

Reply:
Will be resolved in next iteration.